public/v4/apps/langflow.yml

```diff
@@ -0,0 +1,134 @@
+captainVersion: 4
+services:
+    $$cap_appname:
+        image: langflowai/langflow:$$cap_langflow_version
+        volumes:
+            - $$cap_appname-data:/data/langflow
+        restart: always
+        environment:
+            LANGFLOW_SUPERUSER: $$cap_langflow_superuser
+            LANGFLOW_SUPERUSER_PASSWORD: $$cap_langflow_superuser_password
+            LANGFLOW_SECRET_KEY: $$cap_langflow_secret_key
+            LANGFLOW_DATABASE_URL: $$cap_langflow_database_url
+            LANGFLOW_SAVE_DB_IN_CONFIG_DIR: true
+            LANGFLOW_AUTO_LOGIN: $$cap_langflow_auto_login
+            LANGFLOW_BACKEND_ONLY: $$cap_langflow_backend_only
+            LANGFLOW_DEV: $$cap_langflow_dev
+            LANGFLOW_OPEN_BROWSER: $$cap_langflow_open_browser
```

Comment:
Since could be hardcoded to false. Since it's run in a container, it won't launch the browser anyways.

Reply:
yes good point. I'll update that in a little while